src/code_18F340.c

```diff
@@ -17,18 +17,19 @@ ApiStatus func_80260DB8(ScriptInstance* script, s32 isInitialCall) {
 
 ApiStatus func_80260DD8(ScriptInstance* script, s32 isInitialCall) {
     Actor* player = gBattleStatus.playerActor;
+    s32 var;
 
-    if (!gPlayerData.hasActionCommands) {
-        s32 var = player->varTable[0];
+    if (gPlayerData.hasActionCommands) {
+        return ApiStatus_DONE2;
+    }
 
-        if (var >= rand_int(100)) {
-            script->varTable[0] = 1;
-        } else {
-            script->varTable[0] = 0;
-        }
+    var = player->varTable[0];
+    if (var >= rand_int(100)) {
```

Comment:
Why not `if (player->varTable[0] >= rand_int(100))`?

Reply:
becasue it's needed for matching